queries/nginx/highlights.scm

```diff
@@ -0,0 +1,61 @@
+(comment) @comment @spell
+
+(value) @variable
+
+(attribute
+  (keyword) @attribute)
+
+[
+  (location_modifier)
+  "="
+] @operator
```

Comment:
Should `(location_modifier)` be `@keyword.modifier` ?

Reply:
@ObserverOfTime idk, what do you think?:
```js
    location_modifier: (_) =>
      choice(
        "=", // An exact match of the URI path.
        "~", // A case-sensitive regular expression match.
        "~*", //A case-insensitive regular expression match.
        "^~", //Indicates that the following path should be considered the best non-regular expression match.
      ),
```